lib/security/NTLMSecurity.js

```diff
@@ -0,0 +1,32 @@
+"use strict";
+
+var _ = require('lodash');
+
+function NTLMSecurity(username, password, domain, workstation) {
+    if (typeof username == "object") {
+        this.defaults = username;
+        this.defaults.ntlm = true;
+    } else {
+        this.defaults = {
+            ntlm: true,
+            username: username,
+            password: password,
+            domain: domain,
+            workstation: workstation
+        };
+    }
+}
+
+NTLMSecurity.prototype.addHeaders = function (headers) {
+  headers.Connection = 'keep-alive';
```

Comment:
so, is it intentional, that the method `addHeaders` does not add `headers` anywhere but sets a `Connection` property on the (hopefully available) `headers` param (Object)?

Reply:
It looks that way, it's in the original code. It works the same way as in BearerSecurity, which I'd guess is what this code is based on.

Reply:
ah, I see.... hmmm.....